packages/tunnel-client/src/utils/get-local-address.ts

```diff
@@ -0,0 +1,12 @@
+const LOCAL_ADDRESS_ENV_VARIABLE = "METICULOUS_LOCAL_ADDRESS";
```

Comment:
maybe something like this?
```suggestion
const LOCAL_ADDRESS_ENV_VARIABLE = "METICULOUS_TUNNEL_CLIENT_REQEUST_LOCAL_ADDRESS";
```